apps/docs/src/app/platform/api-files.ts

```diff
@@ -9,6 +9,7 @@ export const API_FILES = {
     button: ['ButtonComponent'],
     checkbox: ['CheckboxComponent'],
     checkboxGroup: ['CheckboxGroupComponent'],
+    dynamicPage: ['DynamicPageComponent'],
```

Comment:
added relevant classes. Did not add `DynamicPageFooterComponent,
        DynamicPageGlobalActionsComponent,
        DynamicPageLayoutActionsComponent` since they don't have anything specific they implement.